cloud/jenkins/psmdb_docker_build.groovy

```diff
@@ -105,12 +151,12 @@ pipeline {
                         withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                             sh '''
                                 docker buildx create --use
-                                cd ./source/
                                 sg docker -c "
                                     docker login -u '${USER}' -p '${PASS}'
-                                    RHEL=1 DOCKER_DEFAULT_PLATFORM='linux/amd64,linux/arm64' ./e2e-tests/build
+                                    RHEL=1 DOCKER_DEFAULT_PLATFORM='linux/amd64,linux/arm64' ./source/e2e-tests/build
                                     docker logout
                                 "
+                                echo "perconalab/percona-server-mongodb-operator:main" >> list-of-images.txt
```

Comment:
The same here:
When we run release build we run it from release branch and tag is release-1-20-0 like:
IMAGE=perconalab/percona-server-mongodb-operator:release-1-20-0
We need to take this into account

Reply:
fixed